tutorials/Numerics/TimeStepping/tutorial_risingbubble_config.jl

```diff
@@ -0,0 +1,316 @@
+# # [Rising Thermal Bubble Configuration](@id Rising-Thermal-Bubble-Configuration)
```

Comment:
Is this tutorial meant to be run? Looking at the output, it seems that all of the methods are defined, but never called

Reply:
Ah, nevermind, I see.